README.md

```diff
@@ -55,25 +57,24 @@ _opbeat('setUserContext', {
 </script>
 ```
 
-## Framework specific code instrumentation
+## Framework-specific approach
 
-In order to provide code instrumentation we have chosen a framework-specific approch. This means we'll provide builds of ``opbeat-js`` that include the specific hooks.
+To provide in-depth analysis of your application we have chosen a framework-specific approach. This also means that you only need to load the code that is relevant to your application. You can find framework-specific builds in our release.
```

Comment:
We provide framework specific performance metrics. This means that you need to load opbeat-js build that is relevant to your framework. You can find framework-specific builds in our release